packaging/postinst

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+if [ -f /var/lib/irods/scripts/chown_directories_for_postinstall.py ] ; then
```

Comment:
please add a comment about why this is okay.

e.g. we're only fixing the stomping when the server package is installed, and if it's not, it doesn't matter.